config/rbac/submariner-route-agent/cluster_role.yaml

```diff
@@ -57,3 +57,11 @@ rules:
       - update
     resources:
       - nodes
+  - apiGroups:
+    - projectcalico.org
```

Comment:
I guess you saw, but this needs another level of indentation

```
./config/rbac/submariner-route-agent/cluster_role.yaml
  61:5      error    wrong indentation: expected 6 but found 4  (indentation)
```

Reply:
Yep, updated the PR 
Thanks @dfarrell07